src/SFA.DAS.VacancyServices.Functions/SchedulerFunctions/VacancyIndexScheduler.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.VacancyServices.Functions.Infrastructure;
+
+namespace SFA.DAS.VacancyServices.Functions.SchedulerFunctions
+{
+    public class VacancyIndexScheduler : SchedulerBase
+    {
+        public VacancyIndexScheduler(StorageQueueService service)
+        : base(SchedulerFunctionConstants.QueueNames.VacancyIndexSchedulerQueueName, service)
+        {
+        }
+
+        [FunctionName(nameof(VacancyIndexScheduler))]
+        public async Task Run(
+            [TimerTrigger("%VacancyIndexSchedule%")]TimerInfo myTimer, 
+            ILogger log)
+        {
+            log.LogInformation($"{nameof(VacancyIndexScheduler)} triggered at {DateTime.Now}");
```

Comment:
I would say we can leave this as is, because ideally, the worker that consumes these messages would change to handle an empty message which would enable us to change these functions in the future to simply return string.empty to a storage queue outlined in the output of the function. This would mean we could then delete the base class also and the only remaining code would be the triggers